### PR TITLE
tests: update travis-ci go versions - up to 1.13

### PR DESCRIPTION
**Pull request by ploxiln**

### CODE REVIEW

**Comment by ploxiln** [comment]
cc @jehiah @mreiferson

File: .travis.yml [hunk 1]
```diff
@@ -3,6 +3,8 @@ go:
   - 1.9.x
   - 1.10.x
   - 1.11.x
+  - 1.12.x
```
Comment (jehiah) [inline .travis.yml]: Do we want to switch to just support the last 2 releases here and in go-nsq? (i.e. 1.12.x and 1.13.x)

Comment (ploxiln) [reply .travis.yml]: I like supporting old versions when it is easy. I was thinking about dropping the versions in the middle though (assuming that if 1.9 works and 1.12 works then 1.10 and 1.11 very likely are fine).

Comment (jehiah) [reply .travis.yml]: 👍 Can you capture that as a comment in this file along the lines of "Tests are run against the two most recent Go versions, and a minimum supported version (when different)"

Comment (ploxiln) [reply .travis.yml]: sounds good, done

**Pull request merged**